pakyow-test/lib/test_help/ext/request.rb

```diff
@@ -1,7 +1,9 @@
 module Pakyow
   class Request
-    def params
-      env['pakyow.params']
+    old_params = instance_method(:params)
+
+    define_method :params do
```

Comment:
Fair point. I think personal preference comes into play somewhat since I don't find the alias method approach any easier to understand. Rails made it popular so it's at least familiar.

The problem I have with the alias method chain is that it pollutes the namespace. When you override a method the intent is to replace the existing one. But, the result of `Foo.instance_methods` contains both `old_bar` and `bar`. That bothers me :-/

However, I think we can avoid both of these options and use `Module#prepend`. Pakyow doesn't support Ruby versions prior to 2.0, so we should be good to use it. Just pushed up the change. Let me know what you think!